spec_legacy/support/file_helpers.rb

```diff
@@ -0,0 +1,40 @@
+#-- copyright
+# OpenProject is a project management system.
+# Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See doc/COPYRIGHT.rdoc for more details.
+#++
+
+module FileHelpers
+  module_function
+
+  def mock_uploaded_file(name: 'test.txt',
+                         content_type: 'text/plain',
+                         content: 'test content',
+                         binary: false)
+
+    tmp = ::OpenProject::Files.create_temp_file name: name, content: content, binary: binary
+    Rack::Test::UploadedFile.new tmp.path, content_type, binary
+  end
+end
```

Comment:
As this is copied, I think it makes sense to at least give it a different name to avoid one implementation of FileHelpers overwriting the other one.